src/iptools-jquery-accordion.js

```diff
@@ -92,7 +92,9 @@
     init: function() {
 
       this.$panels = this.$element.children(selectorFromClass(this.settings.panelClass));
-      var $activePanels = this.$panels.filter(selectorFromClass(this.settings.panelActiveClass));
+      var $activePanels = this.$panels
+        .filter(selectorFromClass(this.settings.panelActiveClass))
+        .add(this.$panels.filter('[data-accordion-expanded="true"]'));
```

Comment:
@ain I would call for naming the data attribute "data-panel-active" or "data-accordion-panel-active" as it is only the panel that will be expanded and not the whole accordion and to be consistent with the class naming.